packages/pulpcore/python-requests/python-requests.spec

```diff
@@ -2,14 +2,13 @@
 %global pypi_name requests
 
 Name:           python-%{pypi_name}
-Version:        2.23.0
+Version:        2.24.0
 Release:        1%{?dist}
 Summary:        Python HTTP for Humans
 
 License:        Apache 2.0
 URL:            https://requests.readthedocs.io
 Source0:        https://files.pythonhosted.org/packages/source/r/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
-Patch0:         patch-requests-certs.py-to-use-the-system-CA-bundle.patch
```

Comment:
Is this still needed?

Reply:
yes, patched certifi is here: https://github.com/theforeman/foreman-packaging/pull/5555